psi4/src/psi4/libfock/jk.h

```diff
@@ -483,6 +511,20 @@ class PSI_API JK {
     void set_early_screening(bool early_screening) { early_screening_ = early_screening; }
     bool get_early_screening() { return early_screening_; }
 
+    /**
+     * Set whether or not to perform incremental Fock build?
+     */
+    bool set_incfock(bool incfock) { incfock_ = incfock; }
+    /**
+     * Returns whether or not an incremental Fock build is performed in an SCF iteration
```

Comment:
```suggestion
     * Returns whether or not an incremental Fock build is performed during this SCF iteration
```

Reply:
is or was?

Reply:
You raise a good point! "Was" is probably the better term to use here.

Reply:
Done